tyr-core/src/main/java/org/jboss/tyr/ci/TeamCity.java

```diff
@@ -81,6 +82,13 @@ public void init() {
         this.jsonb = JsonbBuilder.create();
     }
 
+    @PreDestroy
```

Comment:
I just wonder if this shouldn't rather be in the explicit close method since the client is created in explicit init method but we can change that later if needed.